src/store.ts

```diff
@@ -114,7 +114,7 @@ export function useStore(selector, options: SelectorOptions = {}): any {
     updateFunctions.forEach(function(updateFn) {
```

Comment:
we should use a for loop here. foreach + function is much slower, and this is a hot code path